app/models/ncr/work_order.rb

```diff
@@ -134,7 +134,7 @@ def not_ba60?
     def total_price
       amount || 0.0
     end
-
+    
```

Comment:
heh I didn't realize we were already normalizing on `total_price`

let's pick one or the other. creating new clients will make this a PITA to keep sync'd.